docs/community-clients/index.asciidoc

```diff
@@ -1,223 +1,230 @@
-
-* https://github.com/humangeo/rawes[rawes]:
-  Python low level client.
 
-* https://github.com/mozilla/elasticutils/[elasticutils]:
-  A friendly chainable Elasticsearch interface for Python.
+[[clojure]]
+== Clojure
 
-* http://intridea.github.io/surfiki-refine-elasticsearch/[Surfiki Refine]:
-  Python Map-Reduce engine targeting Elasticsearch indices.
+* http://github.com/clojurewerkz/elastisch[Elastisch]:
+  Clojure client.
 
-* http://github.com/aparo/pyes[pyes]:
-  Python client.
+[[cold-fusion]]
+== Cold Fusion
 
+The following project appears to be abandoned:
 
-[[community-ruby]]
-=== Ruby
+* https://github.com/jasonfill/ColdFusion-ElasticSearch-Client[ColdFusion-Elasticsearch-Client]
+  Cold Fusion client for Elasticsearch
 
-See the {client}/ruby-api/current/index.html[official Elasticsearch Ruby client].
+[[erlang]]
+== Erlang
 
-* http://github.com/karmi/retire[Retire]:
-  Ruby API & DSL, with ActiveRecord/ActiveModel integration (retired since Sep 2013).
+* http://github.com/tsloughter/erlastic_search[erlastic_search]:
+  Erlang client using HTTP.
 
-* https://github.com/PoseBiz/stretcher[stretcher]:
-  Ruby client.
+* https://github.com/dieswaytoofast/erlasticsearch[erlasticsearch]:
+  Erlang client using Thrift.
 
-* https://github.com/wireframe/elastic_searchable/[elastic_searchable]:
-  Ruby client + Rails integration.
+* https://github.com/datahogs/tirexs[Tirexs]:
+  An https://github.com/elixir-lang/elixir[Elixir] based API/DSL, inspired by
+  http://github.com/karmi/tire[Tire]. Ready to use in pure Erlang
+  environment.
 
-* https://github.com/ddnexus/flex[Flex]:
-  Ruby Client.
 
-* https://github.com/printercu/elastics-rb[elastics]:
-  Tiny client with built-in zero-downtime migrations and ActiveRecord integration.
+[[go]]
+== Go
 
-* https://github.com/toptal/chewy[chewy]:
-  Chewy is ODM and wrapper for official elasticsearch client
+* https://github.com/mattbaird/elastigo[elastigo]:
+  Go client.
 
-* https://github.com/ankane/searchkick[Searchkick]:
-  Intelligent search made easy
+* https://github.com/belogik/goes[goes]:
+  Go lib.
 
+* https://github.com/olivere/elastic[elastic]:
+  Elasticsearch client for Google Go.
 
-[[community-php]]
-=== PHP
 
-See the {client}/php-api/current/index.html[official Elasticsearch PHP client].
+[[groovy]]
+== Groovy
 
-* http://github.com/ruflin/Elastica[Elastica]:
-  PHP client.
+See the {client}/groovy-api/current/index.html[official Elasticsearch Groovy client].
 
-* http://github.com/nervetattoo/elasticsearch[elasticsearch] PHP client.
+[[haskell]]
+== Haskell
+* https://github.com/bitemyapp/bloodhound[bloodhound]:
+  Haskell client and DSL.
 
-* http://github.com/polyfractal/Sherlock[Sherlock]:
-  PHP client, one-to-one mapping with query DSL, fluid interface.
 
-* https://github.com/nervetattoo/elasticsearch[elasticsearch]
-  PHP 5.3 client
+[[java]]
+== Java
 
-[[community-java]]
-=== Java
+See the {client}/java-api/current/index.html[official Elasticsearch Java clients].
```

Comment:
Clients or client? I realize there is transport vs. node, but not sure how that plays in here.